src/main/java/com/parasoft/demoapp/service/GlobalPreferencesDefaultSettingsService.java

```diff
@@ -283,4 +288,12 @@ public String defaultOrderServiceDestinationQueue() {
     public String defaultOrderServiceReplyToQueue() {
         return ActiveMQConfig.DEFAULT_QUEUE_INVENTORY_RESPONSE;
     }
+
+    public String defaultOrderServiceRequest() {
```

Comment:
Rename it to `defaultOrderServiceRequestTopic()` and rename `defaultOrderServiceResponse()` to `defaultOrderServiceResponseTopic()`.